wf-federated/config.yaml

```diff
@@ -0,0 +1,5 @@
+urls:
+  finlad: "https://csc.fi/o/csc-theme/images/csc-logo-teksti-fi.png"
+  greece: "https://gsri.gov.gr/wp-content/uploads/2022/05/Athena_Horizontal-Ext_EN-page-001-3.jpg"
+  czech: "https://www.mzv.cz/public/a8/13/d5/661666_637430_mzv_en.png"
```

Comment:
I think we should perhaps rather write `czech_republic`. People sometimes feel ways about things like these, and for the other countries, we use the proper country name and not the adjective.

Reply:
done.